ui/src/app/components/target-details-form/target-details-form.component.ts

```diff
@@ -1,15 +1,50 @@
 import { Component, Inject, OnInit } from '@angular/core'
 import { FormBuilder, FormGroup, Validators } from '@angular/forms'
 import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog'
-import { TargetDetails } from 'src/app/app.constants'
-
+import { MigrationDetails, TargetDetails } from 'src/app/app.constants'
 @Component({
   selector: 'app-target-details-form',
   templateUrl: './target-details-form.component.html',
   styleUrls: ['./target-details-form.component.scss'],
 })
 export class TargetDetailsFormComponent implements OnInit {
   targetDetailsForm: FormGroup
+  regionList = [
```

Comment:
Can we get this from an API ?

Reply:
in locations package the API is only available for REST and it requires auth token. Datastream has GetLocations API but it doesn't return the response expected. Will circle back on this later and see if the implementation is done for go locations API